src/main/java/seedu/mtracker/error/ErrorMessage.java

```diff
@@ -71,32 +71,15 @@
     public static final String EMPTY_CURR_PRICE_IN_FILE_ERROR = "Oh no! Looks like the curr price entry in the file"
             + " is empty.";
 
+    public static final String EDIT_EMPTY_ERROR = "Edit parameters cannot be empty, aborting edit process.";
+    public static final String INVALID_EDIT_STATUS = "Sorry status must not be empty"
+            + "and either y/n for done or not done.";

```

Comment:
```suggestion
            + "Enter either done or undone.";
```
y/n doesn't really make sense given that its query abt status